mmv1/third_party/terraform/services/netapp/resource_netapp_storage_pool_test.go

```diff
@@ -0,0 +1,140 @@
+//
+// ----------------------------------------------------------------------------
+//
+//     This file is automatically generated by Magic Modules and manual
+//     changes will be clobbered when the file is regenerated.
+//
+//     Please read more about how to change this file in
+//     .github/CONTRIBUTING.md.
+//
+// ----------------------------------------------------------------------------
+
+package netapp_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccNetappstoragePool_storagePoolCreateExample_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNetappstoragePool_storagePoolCreateExample_full(context),
+			},
+			{
+				ResourceName:            "google_netapp_storage_pool.test_pool",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location", "storage_pool_id"},
+			},
+			{
+				Config: testAccNetappstoragePool_storagePoolCreateExample_update(context),
+			},
+			{
+				ResourceName:            "google_netapp_storage_pool.test_pool",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location", "storage_pool_id"},
+			},
+		},
+	})
+}
+
+func testAccNetappstoragePool_storagePoolCreateExample_full(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+
+resource "google_compute_network" "peering_network" {
+  name = "tf-test-test-network%{random_suffix}"
+}
+
+# Create an IP address
+resource "google_compute_global_address" "private_ip_alloc" {
+  name          = "tf-test-test-address%{random_suffix}"
+  purpose       = "VPC_PEERING"
+  address_type  = "INTERNAL"
+  prefix_length = 16
+  network       = google_compute_network.peering_network.id
+}
+
+# Create a private connection
+resource "google_service_networking_connection" "default" {
+  network                 = google_compute_network.peering_network.id
+  service                 = "netapp.servicenetworking.goog"
+  reserved_peering_ranges = [google_compute_global_address.private_ip_alloc.name]
+}
+
+resource "google_netapp_storage_pool" "test_pool" {
+  storage_pool_id = "tf-test-test-pool%{random_suffix}"
+  location = "us-central1"
+  service_level = "PREMIUM"
+  capacity_gib = "2048"
+  network = google_compute_network.peering_network.id
+  active_directory      = ""
+  description           = ""
+  global_access_allowed = false
+  kms_config            = ""
+  labels                = {}
+  ldap_enabled          = false
+  psa_range             = ""
```

Comment:
let's keep `active_directory` and `kms_config` as empty string, but can we pass in some values for `description`, `psa_range` and `labels`?

Reply:
Added values for description, psa_range and labels.